### DISCUSSION TITLE
脚本到底在那啊？？？

### DISCUSSION THREAD

**Discussion by 502milk** [koishijs/koishi-plugin-booru#87] [General]
![image](https://github.com/koishijs/koishi-plugin-booru/assets/49502047/db53215d-41d9-4a59-b564-ef3aafe4bf48)
不是，文档就写个这个脚本，脚本在哪啊？？指个路好不好啊

**Comment by Lipraty** [comment]
cc @MaikoTan

**Reply by 502milk** [reply]
？？？

**Reply by Lipraty** [reply]
别急，这里的链接没写上去，在帮你 at 应该修这个的人

**Reply by 502milk** [reply]
那就只能等着咯·-·

**Reply by Lipraty** [reply]
开发者也不是 24h 盯着 gh 的。如果很着急的话可以用别的脚本，比如我刚刚找的 [gppt](https://github.com/eggplants/get-pixivpy-token)。

善用搜索，github 内也可以找的

**Reply by 502milk** [reply]
![image](https://github.com/koishijs/koishi-plugin-booru/assets/49502047/78adffb9-1133-4419-9ef5-6ff3d25097c5)
大概...cookie直接扔上去就行？

**Reply by 502milk** [reply]
看来不行...code 400
跑去用那个py的把

**Reply by Lipraty** [reply]
It's better to `refresh token` as the docs. but I don't know the `Pixiv OAuth Script` recommended by MaikoTan.

So, you can get refresh_token from gppt. Or you can wait for MaikoTan to update the documentation and try again.

**Comment by MaikoTan** [comment]
The URL of the script has been proposed in the commit https://github.com/koishijs/koishi-plugin-booru/commit/db261733e46ffabcdc35c30f8aa01e4a8f03a6cd , thanks for the reporting
